### ISSUE TITLE
ruby breaking test on catalina

### ISSUE THREAD

**Issue by eyberg** [nanovms/nanos#1028]
```
also I noticed a problem in the ruby e2e tests under catalina - sorry I forgot to file...basically flags changed for gem changed ("--no-rdoc --no-ri" should change to "-N") ... don't know why they shifted cmd line options around like that
```

**Comment by eyberg** [comment]
think this was fixed @wjhun ?

**Comment by wjhun** [comment]
Yes, thanks. Fixed in #1153.
